pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -136,6 +136,8 @@ stdenv.mkDerivation ({
     "-DCOMPILER_RT_OS_DIR=baremetal"
   ] ++ lib.optionals (stdenv.hostPlatform.isDarwin) (lib.optionals (lib.versionAtLeast release_version "16") [
     "-DCMAKE_LIPO=${lib.getBin stdenv.cc.bintools.bintools}/bin/${stdenv.cc.targetPrefix}lipo"
+  ] ++ lib.optionals (!haveLibcxx) [
+    "-DCOMPILER_RT_HAS_G_FLAG=ON"
```

Comment:
I don’t know whether we’re bootstrapping this the way upstream expects. The goal with these changes is to take advantage of how the LLVM derivation is written in nixpkgs instead of having a bunch of overrides micromanaging the process in the stdenv bootstrap (and then having to fix things LLVM-side anyway for cross-compilation).

Reply:
I added a comment that says (more or less) what the commit message says.